packages/unisim_archisec/unisim_archisec.0.0.10/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "UNISIM-VP DBA decoder"
+description: """
+
+UNISIM ARCHISEC is a companion package of the binary analysis platform
+BINSEC. It exposes disassembly metadata and DBA (Dynamic Bitvector Automata)
+semantics of several instruction set architectures, including ARM, x86_64,
+ppc64 and Sparcv8."""
+maintainer: ["BINSEC <binsec@saxifrage.saclay.cea.fr>"]
+authors: ["Frédéric Recoules" "Yves Lhuillier"]
+license: "BSD-3-Clause"
+homepage: "https://binsec.github.io"
+bug-reports: "mailto:binsec@saxifrage.saclay.cea.fr"
+depends: [
+  "dune" {>= "3.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
  "dune" {>= "3.0"}
```

Whilst the dependency is transitively implicit, we prefer it to be explicitely listed.

Also, the opam-repository will very shortly stop testing for ocaml versions lower than 4.08 so I've put this version as constraint because failures below this version will become silent soon.